packages/extension/__tests__/DaSidebar.js

```diff
@@ -226,6 +226,16 @@ it('should enable submit when input is a valid url', (done) => {
   });
 });
 
+it.only('should disable request input when disableRequest is true', () => {
```

Comment:
I think it's better to write the test case as "it should disable request input while the request is processed" and refactor the test accordingly. This test is very specific to the property `disableRequest`. It doesn't actually check that the property is changed to `true` when the request is sent. This is why I believe you should simulate a user sending the request than checking that the button is disabled

Reply:
Sure, addressed in my latest push.